profile_xf05id1/startup/15-optics.py

```diff
@@ -2,9 +2,24 @@
 from ophyd import Device
 from ophyd import Component as Cpt
 
+class SRXSlitsWB(Device):
+    #THIS IS BROKEN AT EPICS :LEVEL
```

Comment:
We were talking about the nasty things being done to patch over the undulator controls (which SRX / I have been working with @dhidas-bnl-gov on)

This comment was the first pass at doing said reporting.